corehq/apps/reports/static/reports/v2/js/datagrid/reportFilters.js

```diff
@@ -0,0 +1,36 @@
+hqDefine('reports/v2/js/datagrid/reportFilters', [
+    'jquery',
+    'knockout',
+], function (
+    $,
+    ko
+) {
+    'use strict';
+
+    var reportFilter = function (data) {
+        var self = {};
+
+        self.title = ko.observable(data.title);
+        self.name = ko.observable(data.name);
+        self.value = ko.observableArray();
+        self.defaultValue = ko.observable(data.defaultValue);
+        self.placeholder = ko.computed(function () {
+            return gettext("Select") + " " + self.title() + "...";
```

Comment:
Translation would be more robust if it let the translator reorder words: `_.template(gettext("Select <%= title %>"))({ title: self.title()})`